buffer/src/test/java/io/netty/buffer/SlicedByteBufTest.java

```diff
@@ -224,4 +228,26 @@ public void testWriteIso88591CharSequenceExpand() {
     public void testWriteUtf16CharSequenceExpand() {
         super.testWriteUtf16CharSequenceExpand();
     }
+
+    @Test
+    public void ensureWritableWithEnoughSpaceShouldNotThrow() {
+        ByteBuf buf = buffer(1);
```

Comment:
Agreed this test isn't right. I will allocate a new buffer manually here because I care about the size.